package.json

```diff
@@ -67,11 +67,11 @@
     "async": "3.2.4",
     "trim": "0.0.3",
     "set-value": "4.1.0",
-    "postcss": "8.4.20",
+    "postcss": "8.4.25",
```

Comment:
Does this change means that users would need to update postcss/sass/etc too? Or it's just us?

Reply:
i was updating all these packages when trying to get layers working with itwinui-css/itwinui-variables, but then decided not to, mainly because Sass doesn't make it easy.

these are dev dependencies and we deliver compiled CSS, so the user shouldn't need to update any of these. One exception is some postcss plugins might not understand `@layer`, but currently v1 already uses layers and we haven't heard any reports.